orchestra/actions/install.py

```diff
@@ -275,6 +281,22 @@ def _remove_conflicting_files(self):
         )
         self._run_internal_script(script)
 
+    def _collect_times(self):
+        """Returns a dict[path, times], where times is a tuple(atime_ns, mtime_ns)"""
+        times = {}
+        for root, dirnames, filenames in os.walk(f'{self.environment["TMP_ROOT"]}{self.environment["ORCHESTRA_ROOT"]}'):
```

Comment:
`os.path.join`?

Reply:
Does not work because `ORCHESTRA_ROOT` starts with `/`. It's a pretty bad gotcha
```python
>>> os.path.join("/a", "/b")
'/b'
```